### PR TITLE
Infographics by location, step 3: Add streets table and fill it. Remove duplicates

### PR DESCRIPTION
**Pull request by ziv17**

### CODE REVIEW

**Review by BarVolunteering — COMMENTED** [review]
Looks good!
Few comments below.

File: anyway/models.py [hunk 1]
```diff
@@ -944,6 +944,34 @@ def get_id(self):
...
+    def is_active(self):
+        return True
+
+    def is_anonymous(self):
+        return False
+
+    def get_id(self):
+        return self.yishuv_symbol, self.street
+
+
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/models.py:974]: I thought so as I saw it in other classes derived from `Base` in `models`, but maybe I am wrong?

File: anyway/parsers/cbs/executor.py [hunk 2]
```diff
@@ -703,16 +709,19 @@ def get_files(directory):
     return output_files_dict
 
 
-def import_to_datastore(directory, provider_code, year, batch_size):
+def import_to_datastore(
+    directory, provider_code, year, batch_size
+) -> Tuple[int, Dict[int, List[dict]]]:
     """
     goes through all the files in a given directory, parses and commits them
+    Returns No of new items, and new streets dict.
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/parsers/cbs/executor.py:717]: Yes

File: anyway/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (BarVolunteering) [reply anyway/models.py:974]: Probably a mistake, I will fix the other classes.

[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/models.py]: Removed.

File: anyway/parsers/cbs/executor.py [hunk 3]
```diff
@@ -728,12 +737,75 @@ def import_to_datastore(directory, provider_code, year, batch_size):
...
         if "Not found" in str(e):
-            return 0
-        raise (e)
+            return 0, {}
+        raise e
+
+
+def import_streets_into_db():
+    db.session.query(Streets).delete()
+    db.session.commit()
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/parsers/cbs/executor.py]: Good catch.  Removed extra commit.

**Review by BarVolunteering — APPROVED** [review]
Good work!

**Pull request merged**